tools/clang/test/DXC/Passes/DxilFixConstArrayInitializer/negative-index.ll

```diff
@@ -0,0 +1,63 @@
+; RUN: %dxopt %s -hlsl-passes-resume -dxil-fix-array-init -S | FileCheck %s
+
+
+; Original HLSL:
+
+; groupshared float4 w;
+;
+; [numthreads(1, 1, 1)]
+; void b() {
+;   int i = -1;
+;   w[i] = 0;
+; }
+
+
+; CHECK: define void @b
+; CHECK: store float 0
```

Comment:
Is this store still to a `-1` offset? If I'm reading the code right we just don't transform this so it doesn't cause the compiler to have an OOB access.

Reply:
> Is this store still to a `-1` offset? 

Yes.

> If I'm reading the code right we just don't transform this so it doesn't cause the compiler to have an OOB access.

Yes.

Will do.